google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -185,7 +213,19 @@ public Builder(HttpResponse response) {
       // message
       StringBuilder builder = computeMessageBuffer(response);
       if (content != null) {
-        builder.append(StringUtils.LINE_SEPARATOR).append(content);
+        try {
+          // charset
+          charset = response.getContentCharset();
+          String strContent = getContent();
+          if (strContent != null) {
+            builder.append(StringUtils.LINE_SEPARATOR).append(strContent);
+          }
+        } catch (IOException exception) {
+          // it would be bad to throw an exception while throwing an exception
+          exception.printStackTrace();
+        } catch (IllegalArgumentException exception) {
+          exception.printStackTrace();
```

Comment:
ditto with above: libraries shouldn't print

Reply:
@elharo 
the class prints exception stacktrace, trying to be consistant
https://github.com/googleapis/google-http-java-client/blob/8217035a79bfdc0f4a7193293a8583ee831ee25a/google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java#L171

Reply:
Done